package.json

```diff
@@ -63,13 +63,15 @@
         "svgo": "^2.3.0"
     },
     "peerDependencies": {
+        "@react-native-community/picker": "^1.8.1",

```

Comment:
Can't https://www.npmjs.com/package/@react-native-picker/picker be used instead?

![image](https://user-images.githubusercontent.com/3048875/121027420-dbb89780-c79e-11eb-958f-bb87c4236af1.png)

Reply:
(i've seen your comment on the PR, but still)

Reply:
The library being used is a wrapper around that library that supports the same approach for both ios and android: both showing a button that when clicked opens a picker. The base library (react-native-picker/picker) shows only the picker in iOS and does not support changing the styling of the button (in iOS there is no button). Lanwstarted library allows for a simple API for both platforms, allows overriding the picker button and has the iOS picker behaviour (container opening with the picker inside) out-of-the-box.

Reply:
I understand that, my point is: you're including `@react-native-community/picker` which is the deprecated version of `@react-native-picker/picker`. Can't you replaced one with other? (while still using the wrapper `react-native-picker-select`)